spec/active_storage_safe_purge_spec.rb

```diff
@@ -32,6 +32,7 @@ def service
 
   context "when using production service" do
     let(:blob) { local_blob.tap { _1.update!(service_name: "scaleway_production") } }
+    before { stub_request(:any, /.*/).to_return(status: 200, body: "", headers: {}) }
```

Comment:
Ça corrige le warning `Error retrieving instance profile credentials: Failed to open TCP connection to 169.254.169.254:80 (execution expired)` (j'ai pas la `RAILS_MASTER_KEY` en local).